config/nrfconnect/chip-module/Kconfig.defaults

```diff
@@ -288,40 +289,38 @@ config MBEDTLS_ENABLE_HEAP
     default y
 
 config MBEDTLS_HEAP_SIZE
-    default 15360
-
-config MBEDTLS_TLS_LIBRARY
-    default y
+    default 15360 if !CHIP_CRYPTO_PSA
+    default 32768 if CHIP_CRYPTO_PSA
```

Comment:
Have we reported this growth to nrf_security team? I think they should be aware of that and try to optimize that?

Reply:
I need to perform more tests. For now, I realized that 16 kB of Heap is too low to run PSA Crypto, 32kB is the secure value. Probably, it can be reduced right now, but first I would like to check the actual demand.